tools/Setup-Test-Environment.ps1

```diff
@@ -0,0 +1,26 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$service,
+    [string]$keyVault = "AzureSDK-Dev-TestSecrets"
+    )
+
+$secrets = (az keyvault secret list --vault-name $keyVault --output json | ConvertFrom-Json);
```

Comment:
This requires that folks have the azure cli installed. Something we would need to at least call out as a dependency.

Reply:
I'll add a test at the beginning of the script.

Reply:
May also be worth adding a test for az login for folks that don't use the azure cli often.

Reply:
Yeah, I've added `az account show ` call to the beginning of the script that would fail if cli is not logged in.